nemo/collections/tts/helpers/helpers.py

```diff
@@ -26,6 +27,14 @@
 from nemo.utils import logging
 
 
+class OperationMode(Enum):
+    """Training or Inference (Evaluation) mode"""
+
+    training = 0
+    validation = 1
+    infer = 2
```

Comment:
I like the fact that you "unified" OperationMode.

Is it something that will be useful only for TTS?

Reply:
I briefly considered adding it to ModelPT, but decided against as I feel like that should be a team decision.

Reply:
Ok